internal/stationxml/v1.1/decimation.go

```diff
@@ -0,0 +1,42 @@
+package stationxml
+
+// Corresponds to SEED blockette 57.
+type Decimation struct {
+	InputSampleRate Frequency
+	Factor          int32
+	Offset          int32
+	Delay           Float
+	Correction      Float
+}
+
+func (d *Decimation) SetInputSampleRate(rate float64) {
+	d.InputSampleRate = Frequency{
+		Float: Float{
+			Value: rate,
+		},
+	}
+}
+
+func (d Decimation) Copy() Decimation {
+	return d
```

Comment:
This may not make a copy

Reply:
It's fine as a copy given that the struct doesn't hold any slices or pointers. (this code will be replaced in a new PR soon).